### ISSUE TITLE
Preprocessing Data

### ISSUE THREAD

**Issue by Niroi** [isunchy/cuboid_abstraction#5]
Hi, 
I'm currently working on trying to retrain the model on the DFAUST dataset, but I hit a wall since I can't find how exactly the data was preprocessed for the model to use. I've looked into the method that you mention in the paper by Tulsiani (2017) but I can't find where the preprocessed data of Tulsiani align with you provided training dataset.

So I'm asking if you could possibly provide the code that you use for creating the .tfrecords files that you also provide as training data?

**Comment by blurgyy** [comment]
I'm also interested in obtaining the preprocessed data from custom meshes, specifically, how are the point coordinates normalized, and how to obtain the "octree" field inside each .tfrecords file.
